internal/gatewayapi/clienttrafficpolicy.go

```diff
@@ -447,7 +447,7 @@ func (t *Translator) translateClientTrafficPolicyForListener(policy *egv1a1.Clie
 		translatePathSettings(policy.Spec.Path, httpIR)
 
 		// Translate HTTP1 Settings
-		if err = translateHTTP1Settings(policy.Spec.HTTP1, httpIR); err != nil {
+		if err = translateHTTP1Settings(policy.Spec.HTTP1, ptr.Deref(connection, ir.ClientConnection{}).ConnectionLimit, httpIR); err != nil {
```

Comment:
curious why we need ptr.Deref here

Reply:
I'll update it, thanks for the suggestion